catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/searchui/standard/endpoints/MetacardEditEndpoint.java

```diff
@@ -178,7 +175,7 @@ public Response setBinaryAttribute(@Context HttpServletResponse response,
         }
 
         catalogFramework.update(new UpdateRequestImpl(id, metacard));
-        Map<String, Object> responseMap = getResponseMap(attribute,
+        getResponseMap(attribute,
```

Comment:
I think we should talk to whoever made this and see if it was just an oversight on their part. If it needs to be returned, then we should hook it up appropriately and return it. If it doesn't, then we can just delete it.